pipeline/clean/clean-mono.sh

```diff
@@ -74,14 +75,36 @@ zstdmt -dc "${output_prefix}.${lang}.langid.zst" |
 parallel --no-notice --pipe -k -j "${threads}" --block 50M \
   "python3 tools/clean_mono.py -l ${lang} --debug" \
   2>"${output_prefix}.${lang}.clean.debug.txt" |
-zstdmt >"${output_prefix}.${lang}.zst"
+zstdmt >"${output_prefix}.${lang}.rule-based.zst"
 
-test -s "${output_prefix}.${lang}.zst" || exit 1
+test -s "${output_prefix}.${lang}.rule-based.zst" || exit 1
+
+######################################################################
+echo "### Filter by fluency score"
+
+if [ "${fluency_threshold}" == "0" ] || [ "${fluency_threshold}" == "0.0" ]; then
+  echo "Threshold is 0, skipping filtering"
+  cp "${output_prefix}.${lang}.rule-based.zst" "${output_prefix}.${lang}.zst"
+else
+  # the model is 125MB, similar in size to the fastText one, so it's ok to download it here
```

Comment:
@bhearsum do you think it's still worth extracting this download to a separate task? I see the models are 100-200Mb, similar to the fastText ones. Extracting would save on some ingress traffic, but would complicate the pipeline and model updates, similar to the bilceaner ones. The bicleaner models are a lot larger, so it's justified to cache them.

Reply:
I have no objections. IMO the more important downside is that not caching these means that you'll be more susceptible to failures when wherever is hosting them is down.

(And I could be wrong, but it looks like [ingress is free](https://cloud.google.com/network-tiers/pricing))

Reply:
Ok, I'll skip it for now then. Let's if we have issues with that. Github is hosting them, so I guess it should be reliable.

https://github.com/bitextor/monocleaner/blob/5a743407179468c83259999987138d6ae590c687/scripts/monocleaner-download#L32